docs/repo-docs/crafting-your-repository/understanding-your-repository.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: Understanding your repository
+description: Learn how to understand your repository structure using Turborepo.
+---
+
+Turborepo includes tools for understanding your repository structure, that can help you use and optimize your codebase.
+
+## `turbo ls`
+
+To list your packages, you can run `turbo ls`. This will show the packages in your repository and where they're located.
+
+```bash title="Terminal"
+> turbo ls
+turbo 2.1.3
+
+ WARNING  ls command is experimental and may change in the future
```

Comment:
```suggestion
 WARNING the `ls` command is experimental and may change in the future
```

Reply:
This is based off the warning output, so I'll keep it as is for now